kit/src/elements/textarea/mod.rs

```diff
@@ -42,6 +44,10 @@ pub struct Props<'a> {
     onreturn: EventHandler<'a, (String, bool, Code)>,
     #[props(!optional)]
     reset: Option<UseState<bool>>,
+    #[props(optional)]
+    is_disabled: Option<bool>,
+    #[props(optional)]
+    tooltip: Option<String>,
```

Comment:
the `#[props(optional)]` has no effect on a `Option`. I recommend not using it for this purpose. 

By default, making any Dioxus prop an `Option` means that if the field is omitted, it defaults to `None`. 
If for some reason you need to pass an actual `Option` to a prop, that macro can be helpful if used like this: `#[props(!optional)]` .

Reply:
oh ok.
i made to field an Option so it doesnt require major refactoring on the things calling the textarea.
will remove the `#[props(optional)]` though

Reply:
replaced Option with props(default...)